packages/react-instantsearch/src/connectors/connectRefinementList.test.js

```diff
@@ -285,7 +294,7 @@ describe('connectRefinementList', () => {
     expect(searchState).toEqual({another: {searchState: 'searchState'}});
   });
 
-  it('calling searchForFacetValues return the right searchForFacetValues parameters', () => {
+  it('calling searchForItems return the right searchForItems parameters', () => {
     const parameters = searchForFacetValues({attributeName: 'ok'}, {}, 'yep');
```

Comment:
Nevermind, we agreed on keep the name `searchForFacetValues` internally in connectors